### ISSUE TITLE
Piano-roll CSS grid-options for beat and tick

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#2832]
One thing about the grid, i always felt: The *difference* between Beat & Tick **dividers**, (the lines) are imo not 'clear', and distinctness of these lines are even more difficult in higher magnification, where the lines really are not changing in width:
![lmmsscaling](https://cloud.githubusercontent.com/assets/6662066/15891626/cf666e58-2d75-11e6-95db-2fdb5b8a0f62.png)
Here scaling to 200 % is shown in gimp to tr. and lmms 200% to tl.
Ofcause lmms will only scale horizontally, they keyboard is the reason for that, but the divider-lines are not scaled, and tick and beat dividers are very similar, as you can see in the screenie.

I have used CSS and it work better with this alpha-220 setup
![cssenhgrid](https://cloud.githubusercontent.com/assets/6662066/15929487/9257265a-2e4d-11e6-9040-d82689a356a0.png)

But the Beat & Tick-dividers are still very similar 

If CSS could give an option like:
![cssdevidergrid](https://cloud.githubusercontent.com/assets/6662066/15929908/20631aec-2e50-11e6-9a13-2fb781b9168e.PNG)
It would be a huge help.
The mockup is ofcause ugly.

What are everyones opinion on difference between beat and tick dividers?

Also workflow could be enhanced with #2308

What are everyones thoughts on divider-line differences?

Another thing that will enhance the workflow in piano-roll, is alternate shading of bars, as known in song-editor

**Comment by Spekular** [comment]
I think dashed and/or dotted lines might work well.

**Comment by musikBear** [comment]
> I think dashed and/or dotted lines might work well.

Yes dashed for ticks could perhaps give enough difference between tick and beat divider-lines. I hope im not the only one that feel that the difference is difficult to see.
